indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -378,6 +378,9 @@
           :science-keywords science-keywords-field-mapping
           :science-keywords.humanized science-keywords-field-mapping
 
+          :granule-data-format                  (m/stored m/string-field-mapping)
+          :granule-data-format.humanized        (m/stored m/string-field-mapping)
```

Comment:
We don't need to store humanized field. Change this to `m/string-field-mapping`.